### PR TITLE
add more philosophy quotes

### PR DESCRIPTION
**Pull request by Francesca-Belfiore**

### CODE REVIEW

File: js/quotes.js [hunk 1]
```diff
@@ -1637,5 +1637,24 @@ const quotes = [
 	{
 		text: "Children are not colouring books. You don't get to fill them with your favourite colours.",
 		author: "Khaled Hosseini",
+	},
+  		text: "It's not what happens to you, but how you react to it that matters.",
```
Comment (castanedadev-edu) [inline js/quotes.js:1642]: Hi! left curly (open) bracket missing here. 
```suggestion
{
  		text: "It's not what happens to you, but how you react to it that matters.",
```

Comment (Francesca-Belfiore) [reply js/quotes.js:1642]: ops forgot it when I resolved the conflict! thanks!

**Comment by seths10** [comment]
Thanks for contributing,  please consider giving the repo a star and also don't forget to follow me

**Comment by Francesca-Belfiore** [comment]
Sure, thank you!

**Pull request merged**